scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TFScioContextFunctions.scala

```diff
@@ -52,6 +52,7 @@ class TFScioContextFunctions(val self: ScioContext) extends AnyVal {
    * remotely stored `org.tensorflow.metadata.v0.Schema` object available in a DistCache.
    * @group input
    */
+  @deprecated("Schema inference should be done using TensorFlow Data Validation", "Scio 0.7.0")
```

Comment:
@brianmartin I would be a bit more specific here and actually mention that this method will be removed.

Reply:
👍 done